config/initializers/errors.rb

```diff
@@ -15,20 +15,20 @@
   Net::HTTPBadResponse,
   Net::HTTPHeaderSyntaxError,
   Net::ProtocolError,
-  Timeout::Error,
+  Timeout::Error
```

Comment:
We do want to have trailing commas. Not sure what the rule for Rubocop is. Might be:
```
Style/TrailingCommaInLiteral:
  EnforcedStyleForMultiline: consistent_comma
```